src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -68,4 +71,33 @@ public void execute_listIsFiltered_showsVideos() {
         String expectedString = String.format(ListCommand.MESSAGE_SUCCESS_VIDEOS, moduleCode, lectureName);
         assertCommandSuccess(listCommand, model, expectedString, expectedModel);
     }
+
+    @Test
+    public void execute_listIsFiltered_moduleNotFound() {
+        ModuleCode moduleCode = new ModuleCode(VALID_MODULE_CODE_2103);
+        String input = String.format("list /mod %s", moduleCode);
+        try {
+            listCommand = new ListCommandParser().parse(input);
+            listCommand.execute(expectedModel);
+        } catch (ParseException e) {
+            assertNull(e);
+        }
+        String expectedString = String.format(MESSAGE_MODULE_DOES_NOT_EXIST, moduleCode);
+        assertCommandSuccess(listCommand, model, expectedString, expectedModel);
+    }
+
+    @Test
+    public void execute_listIsFiltered_lectureNotFound() {
+        ModuleCode moduleCode = module.getCode();
+        LectureName lectureName = new LectureName("Unknown lecture");
+        String input = String.format("list /mod %s /lec %s", moduleCode, lectureName);
+        try {
+            listCommand = new ListCommandParser().parse(input);
+            listCommand.execute(expectedModel);
+        } catch (ParseException e) {
+            assertNull(e);
+        }
+        String expectedString = String.format(MESSAGE_LECTURE_DOES_NOT_EXIST, lectureName, moduleCode);
+        assertCommandSuccess(listCommand, model, expectedString, expectedModel);
```

Comment:
Any reason why the command is successful in this scenario? Shouldn't a `CommandException` be thrown?